lib/bdev/nvme/blockdev_nvme.c

```diff
@@ -502,16 +503,59 @@ probe_cb(void *cb_ctx, const struct spdk_nvme_transport_id *trid,
 }
 
 static void
-blockdev_nvme_timeout_cb(void *cb_arg, struct spdk_nvme_ctrlr *ctrlr,
-			 struct spdk_nvme_qpair *qpair, uint16_t cid)
+abort_cb(void *arg, const struct spdk_nvme_cpl *cpl)
+{
+	/*
+	 * abort completion callback function. Currently, there is nothing to do
+	 * struct spdk_nvme_ctrlr *ctrlr = arg;
+	 */
+}
+
+static void
+blockdev_nvme_timeout_cb(struct spdk_nvme_ctrlr *ctrlr,
+			 struct spdk_nvme_qpair *qpair,
+			 void *cb_arg,
```

Comment:
The timeout_count here is probably not the right way to present this. Instead, a command should only ever time out one time. On the first timeout, the user may elect to send an abort for that command. If that abort times out, this callback will be called for the abort command, and in response to a timeout of an abort command (or any ADMIN command really), a reset should be performed instead of sending another abort.